frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass

```diff
@@ -1,16 +1,17 @@
-@import '../../../../../styles/color-variables'
-@import '../../../../../styles/screen-variables'
+@import 'src/styles/color-variables'
+@import 'src/styles/screen-variables'
+@import url('https://rsms.me/inter/inter.css')
```

Comment:
why did you import this here?

Reply:
It works only with font added locally in sass file